src/app.ts

```diff
@@ -50,14 +49,15 @@ const discordFaucetApp = async (appCred: DiscordCredentials) => {
         throw new Error('No seed phrase was provided for the faucet account');
     }
 
+    const network = Network.shibuya;
```

Comment:
Not clean to hard-code in a network connection on the entry point.

Reply:
Ok, fixed.